src/entities/communitySbt.ts

```diff
@@ -189,15 +224,167 @@ class SBT {
+            for (const badge of subgraphBadges) {
+                badgesResponse.push(badge);
+            }
+            for (const badgeVariant of NON_SUBGRAPH_BADGES_SEASONS[seasonId]) {
+                if (TOP_BADGES_VARIANT[badgeVariant]) {
+                    const topBadge = await this.getTopTraderBadge(subgraphUrl, userId, seasonId, TOP_BADGES_VARIANT[badgeVariant]);
+                    if (topBadge) {
+                        badgesResponse.push(topBadge);
+                    }
+                } else if (NON_PROGRAMATIC_BADGES_VARIANT[badgeVariant] && nonProgBadges[badgeVariant]) {
+                    const nonProgBadge = nonProgBadges[badgeVariant];
+                    badgesResponse.push(nonProgBadge);
+                }
+            }
+            return badgesResponse;
+        } catch (error) {
+          return [];
+        }
+    }
+
+    public async getTopTraderBadge(
+        subgraphUrl: string,
+        userId: string,
+        seasonId: number,
+        badgeType: string
+      ): Promise<BadgeResponse | undefined> {
+        if (!process.env.REACT_APP_SUBGRAPH_BADGES_URL) {
+            return undefined;
+          }
+        try {
+            const badgeQuery = `
+                query( $id: BigInt, $unit: String) {
+                    seasonUsers(first: 5, orderBy: $unit, orderDirection: desc) {
+                        id
+                        badges {
+                        id
+                        awardedTimestamp
+                        mintedTimestamp
+                        badgeType
+                        badgeName
+                        }
+                        totalNotionalTraded
+                        totalLiquidityProvided
+                    }
+                }
+            `;
+            const client = new ApolloClient({
+                cache: new InMemoryCache(),
+                link: new HttpLink({ uri: subgraphUrl, fetch })
+            })
+            const id = `${userId.toLowerCase()}#${seasonId}`;
+            const isNotional = badgeType.indexOf("trade") > -1;
+            const unit = isNotional ? "totalNotionalTraded" : "totalLiquidityProvided";
+            const data = await client.query({
+                query: gql(badgeQuery),
+                variables: {
+                    id: id,
+                    unit: unit
+                },
+            });
+    
+            if (!data.data.seasonUsers) {
+                return undefined;
+            }
+
+            for (const seasonUser of data.data.seasonUsers) {
+                if (seasonUser.id === userId.toLowerCase()) {
+                    const badge : BadgeResponse = {
+                        id: `${userId}#${seasonId}#${badgeType}`,
+                        badgeType: badgeType,
+                        badgeName: TOP_BADGES_VARIANT[badgeType],
+                        awardedTimestamp: isNotional ? seasonUser.notionalAwardedTimestamp : seasonUser.liquidityAwardedTimestamp,
```

Comment:
can the SDK return miliseconds to the FE? 
```
/**
 * "Convert seconds to milliseconds, but only if the input is a number and not zero."
 *
 * @param {number} seconds - number - The number of seconds to convert to milliseconds.
 * @returns A function that takes a number and returns a number or undefined.
 */
export const toMillis = (seconds: number): number | undefined => {
  if (isNaN(seconds) || seconds === 0) {
    return undefined;
  }

  return seconds * 1000;
};
```

Reply:
now timestamps are in ms and named as "awardedTimestampMs" and "mintedTimestampMs" 6d9b2f701490ef8604232b7b48f1621d840e6112